python/cudf/cudf/_libxx/slice.pyx

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
```

Comment:
Don't need to define these anymore, these are now compiler directives in the setup.py